storyTwo.php

```diff
@@ -0,0 +1,63 @@
+<?php
+require 'functions.php';
+
+if (isset($_POST['submit'])) {
+    $db = connectToDB();
+    $errors = validate($_POST, $db);
+
+    if (insertNewCar($errors, $_POST, $db)) {
+        header('Location: carsCollection.php');
+        exit;
+    }
+}
+?>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- CSS -->
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href=" css/style.css">
+    <!-- FONT -->
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,400;0,600;0,700;0,800;1,400&display=swap"
+          rel="stylesheet">
+    <!-- PAGE TITLE -->
+    <title>Cars Collector</title>
+</head>
+<body>
+<section>
+    <div class="title">
+        <p>Choose your car</p>
+    </div>
+</section>
+<section class="form-section">
+    <form action="storyTwo.php" method="POST">
+        <label>Make: </label><br>
+        <input type="text" name="make" value="<?php echo htmlspecialchars($_POST['make'] ?? '') ?>"><br>
+        <div class="text"><?php echo $errors['make'] ?? ''; ?></div>
+        <label>Model: </label><br>
+        <input type="text" name="model" value="<?php echo htmlspecialchars(($_POST['model'] ?? '')) ?>"><br>
```

Comment:
`<br>` tags aren't best practice, please create this space using CSS, instead (you could you use your `.text` class for this)